File: samples/snippets/csharp/VS_Snippets_Remoting/RemotingConfiguration_GetRegisteredWellKnownClient_Client/CS/RemotingConfiguration_GetRegisteredWellKnownClient_Server.cs
using System;
using System.Runtime.Remoting;
using System.Runtime.Remoting.Channels;
using System.Runtime.Remoting.Channels.Tcp;

namespace RemotingSamples 
{
   public class Sample
   {
      public static void Main()
      {
         TcpChannel chan = new TcpChannel(8085);
         ChannelServices.RegisterChannel(chan);
         RemotingConfiguration.RegisterWellKnownServiceType(
           typeof(MyServerImpl),"SayHello", WellKnownObjectMode.Singleton);
         Console.WriteLine("Press <enter> to exit...");
         Console.ReadLine();
      }
   }
}

File: samples/snippets/cpp/VS_Snippets_Remoting/RemotingConfiguration_GetRegisteredWellKnownClient_Client/CPP/remotingconfiguration_getregisteredwellknownclient_server.cpp


#using <System.Runtime.Remoting.dll>
#using <System.dll>
#using <RemotingConfiguration_GetRegisteredWellKnownClient_Shared.dll>

using namespace System;
using namespace System::Runtime::Remoting;
using namespace System::Runtime::Remoting::Channels;
using namespace System::Runtime::Remoting::Channels::Tcp;
int main()
{
   TcpChannel^ chan = gcnew TcpChannel( 8085 );
   ChannelServices::RegisterChannel( chan );
   RemotingConfiguration::RegisterWellKnownServiceType( MyServerImpl::typeid, "SayHello", WellKnownObjectMode::Singleton );
   Console::WriteLine( "Press <enter> to exit..." );
   Console::ReadLine();
}
